dbms/src/Common/CPUAffinityManager.h

```diff
@@ -123,9 +126,18 @@ class CPUAffinityManager
     CPUAffinityManager();
     // Disable copy and move
 public:
-    CPUAffinityManager(const CPUAffinityManager &) = delete;
-    CPUAffinityManager & operator=(const CPUAffinityManager &) = delete;
-    CPUAffinityManager(CPUAffinityManager &&) = delete;
-    CPUAffinityManager & operator=(CPUAffinityManager &&) = delete;
+#define DISALLOW_COPY_AND_MOVE(ClassName) \
+    DISALLOW_COPY(ClassName);             \
+    DISALLOW_MOVE(ClassName)
+
+#define DISALLOW_COPY(ClassName)           \
```

Comment:
You can write the code in /dbms/src/Common/nocopyable.h

Reply:
You can replace it directly. Maybe you can list some of the classes need to refactor in the issue and create some sub-issue, and you refactor 3 or 4 of them in one pr.

Reply:
Like @ywqzzy said, the macro `DISALLOW_COPY`/`DISALLOW_MOVE `/`DISALLOW_COPY_AND_MOVE` should be defined in a common place like `dbms/src/Common/noncopyable.h`.

And I wonder whether we can define `DISALLOW_COPY_AND_MOVE` before `DISALLOW_COPY`/`DISALLOW_MOVE `?